app/javascript/packages/verify-flow/index.tsx

```diff
@@ -10,5 +10,12 @@ interface VerifyFlowProps {
 }
 
 export function VerifyFlow({ initialValues = {} }: VerifyFlowProps) {
-  return <FormSteps steps={STEPS} initialValues={initialValues} promptOnNavigate={false} />;
+  return (
+    <FormSteps
+      steps={STEPS}
+      initialValues={initialValues}
+      promptOnNavigate={false}
+      basePath="/verify/v2/"
```

Comment:
Should find a way to avoid hard-coding this.

Reply:
Add a `data-base-path` attribute from Rails?

Reply:
I'm wondering if there might be a way to just pull the last part of the path fragment, i.e.

```ts
getStep('/path/to/verify/personal_key') === 'personal_key'
```

One challenge is making sure that...

```ts
getStep('/path/to/verify') !== 'verify'
```

A thought is to validate against the list of steps, and default to the first step if it's not one of the known step names.

Reply:
> A thought is to validate against the list of steps, and default to the first step if it's not one of the known step names.

Actually, it already works this way by default!

https://github.com/18F/identity-idp/blob/9c8739525ecbcebca133774ebba1275487242ffc/app/javascript/packages/form-steps/form-steps.tsx#L192-L193

Reply:
In the end, I went with a data attribute, since while we could reliably determine the current step on first page render, we couldn't reliably _add_ the step to the current URL without knowing the base path.

Updated in ce7c0d8a6.